downstream/modules/platform/proc-controller-set-up-LDAP.adoc

```diff
@@ -0,0 +1,174 @@
+[id="controller-set-up-LDAP"]
+
+= Setting up LDAP authentication
+
+When configured, a user who logs in with an LDAP username and password automatically gets an {ControllerName} account created for them and they can be automatically placed into organizations as either regular users or organization administrators.
+
+Users created through an LDAP login cannot change their username, first name, last name, or set a local password for themselves. 
+You can also configure this to restrict editing of other field names.
+
+[NOTE]
+====
+If the LDAP server you want to connect to has a certificate that is self-signed or signed by a corporate internal certificate authority (CA), 
+you must add the CA certificate to the system's trusted CAs. 
+Otherwise, connection to the LDAP server results in an error that the certificate issuer is not recognized. 
+For more information, see link:https://access.redhat.com/solutions/5136201[How Do I Import CA Certificate (openldap, AD, IDM, FreeIPA) Required for LDAPS Integration?]. 
+If prompted, use your Red Hat customer credentials to login.
+====
+
+.Procedure
+
+. Create a user in LDAP that has access to read the entire LDAP structure.
+. Use the `ldapsearch`` command to test if you can make successful queries to the LDAP server.
+This is a command line tool that you can install on the {ControllerName} system's command line as well as on other Linux and OSX systems.
```

Comment:
You can install this tool fom {ControllerNames}'s system command line, and using other Linux and OSX systems.